src/jobs/jobs.controller.ts

```diff
@@ -588,53 +642,50 @@ export class JobsController {
     type: JobClass,
     description: "Created job",
   })
-  async create(
+  async createV3(
     @Req() request: Request,
     @Body() createJobDto: CreateJobDto,
```

Comment:
Testing this more I noticed a bug. The v3 endpoint is using the v4 DTO (datasets in the jobParams).